CONTRIBUTING.md

```diff
@@ -74,7 +74,19 @@ The followings are required for developing TiDB Dashboard:
    pnpm dev
    ```
 
-1. That's it! You can access TiDB Dashboard now: http://127.0.0.1:3001
+4. That's it! You can access TiDB Dashboard now: http://127.0.0.1:3001
+
+5. (Optional) Package frontend and backend into a single binary:
+
+   ```bash
+   # In tidb-dashboard directory:
+   make package
+   
+   # Run the binary without separate frontend server:
+   make run
+   ```
+
+   You can access TiDB Dashboard now: [http://127.0.0.1:12333](http://127.0.0.1:3001)
```

Comment:
Should be `[http://127.0.0.1:12333/dashboard](http://127.0.0.1:12333/dashboard)`?